INN/umbrella-ipbs
# Styling for homepage events calendar section

**Drafted until header and footer are merged** 
## Changes

This pull request makes the following changes:

- Adds styling for the homepage events section

Default:
![Screen Shot 2020-01-27 at 4 25 15 PM](https://user-images.githubusercontent.com/18353636/73216163-7f158900-4123-11ea-8137-ef1066faecc6.png)

Details hover:
![Screen Shot 2020-01-27 at 4 29 20 PM](https://user-images.githubusercontent.com/18353636/73216159-7f158900-4123-11ea-9940-ff722ec20449.png)

Mobile:
![Screen Shot 2020-01-27 at 4 25 22 PM](https://user-images.githubusercontent.com/18353636/73216161-7f158900-4123-11ea-9434-b45ec4a2b3ea.png)

## Why



For #42 

## Testing/Questions

Features that this PR affects:

- Homepage events section


Questions that need to be answered before merging:


Steps to test this PR:

1. Install the `Simple Calendar – Google Calendar Plugin` plugin
2. Under `Calendars -> Settings` add your Google Calendar API key (test one I created is `AIzaSyBBm--ABBcu-VoPm8q04IoWYZcfqqNjoWY`)
3. Add a new calendar with this as the content:
```
<div class="date">[when]</div>
<h4>[title]</h4>
<div class="description">[description]</div>
<div class="details"><div class="details-link">[link newwindow="yes"]EVENT DETAILS[/link]</div></div>
```
4. Use these settings:
![Screen Shot 2020-01-27 at 4 36 27 PM](https://user-images.githubusercontent.com/18353636/73216072-4aa1cd00-4123-11ea-82ee-86ad43618d23.png)
![Screen Shot 2020-01-27 at 4 36 36 PM](https://user-images.githubusercontent.com/18353636/73216071-4aa1cd00-4123-11ea-835b-dd209fb48b2c.png)
![Screen Shot 2020-01-27 at 4 36 49 PM](https://user-images.githubusercontent.com/18353636/73216069-4a093680-4123-11ea-8a21-241a9773f0e3.png)

5. Add the ID of a google calendar to test with. The one I created and have been using is `investigativenewsnetwork.org_of085evtehdohnvu2q4qc6v9pc@group.calendar.google.com`
6. Add the `Simple Calendar` widget to the homepage bottom widget area with `Upcoming Events` as the title and select the test calendar you created
7. View on the frontend and verify it looks ok
